sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -378,11 +377,133 @@ export function toAmqpMessage(msg: SendableMessageInfo): AmqpMessage {
   return amqpMsg;
 }
 
+/**
+ * A message that can be settled by completing it, abandoning it, deferring it, or sending
+ * it to the dead letter queue.
+ */
+export interface ReceivedSettleableMessage extends ReceivedMessage {
```

Comment:
I would suggest to move the `ReceivedSettleableMessage` declaration to appear after the one for `ReceivedMessage` for ease of readability as the former extends the latter

Reply:
Done.